sos/upload/__init__.py

```diff
@@ -0,0 +1,161 @@
+# Copyright 2024 Red Hat, Inc. Jose Castillo <jcastillo@redhat.com>
+
+# This file is part of the sos project: https://github.com/sosreport/sos
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# version 2 of the GNU General Public License.
+#
+# See the LICENSE file in the source distribution for further information.
+
+import os
+from textwrap import fill
+from sos.component import SoSComponent
+from sos import _sos as _
+from sos import __version__
+
+
+class SoSUpload(SoSComponent):
+    """
+    This class is designed to upload files to a distribution
+    defined location. These files can be either sos reports,
+    sos collections, or other kind of files like: vmcores,
+    application cores, logs, etc.
+
+    """
+
+    desc = """
+            Upload a file (can be a sosreport, a must-gather, or others) to
```

Comment:
`sos report`?

Reply:
oh wow, how could I forget about this...
Fixed in the next push